youtube_dl/extractor/tunepk.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+
+class TunepkIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://|\.)(tune\.pk)/(?:player|video|play)/(?:[\w\.\?]+=)?(?P<id>\d+)'
+    _TEST = {
+        'url': 'https://tune.pk/video/6919541/maudie-2017-international-trailer-1-ft-ethan-hawke-sally-hawkins',
+        'md5': '0C537163B7F6F97DA3C5DD1E3EF6DD55',
+        'info_dict': {
+            'id': '6919541',
+            'ext': 'mp4',
+            'title': 'Maudie (2017) | International Trailer # 1 ft Ethan Hawke, Sally Hawkins',
+            'thumbnail': r're:^https?://.*\.jpg$'
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        html = webpage.replace('\n\r', '').replace('\r', '').replace('\n', '').replace('\\', '')
+        sources = re.compile('"sources"\s*:\s*\[(.+?)\]').findall(html)[0]
```

Comment:
Use `self._parse_json` instead. This also looks very similar to `sources` parsing in `self._parse_jwplayer_data` that should be probably generalized.